### PR TITLE
Glossary replacement

### PR DESCRIPTION
**Pull request by devinmarieb**
# Description
- Replacing the glossary.html with a glossary.adoc
- Archiving the old glossary folder/files
- Removing the glossary build from the config

# Reasons
There was no reason for this page to be different from the others.
[Jira Ticket](https://circleci.atlassian.net/jira/software/projects/DOCSTEAM/boards/412?selectedIssue=DOCSTEAM-240)

### CODE REVIEW

**Comment by rsylvian** [comment]
Thanks @devinmarieb! We are going to schedule this for our Next sprint (which starts Wednesday)

File: jekyll/_cci2/glossary.adoc [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+<<orb-concepts#,Orbs>> are reusable snippets of code that help automate repeated processes, accelerate project setup, and make it easy to integrate with third-party tools. Visit the https://circleci.com/developer/orbs[Orbs Registry] on the CircleCI Developer Hub to search for orbs to help simplify your config.
+
+== Owner
+
+The owner of the team's VCS organization.
+
+== Pipeline
+
+A <<pipelines#,CircleCI pipeline>> is the full set of processes you run when you trigger work on your projects. Pipelines encompass your workflows, which in turn coordinate your jobs. Pipelines are defined in your project '.circleci/config.ym.' file. _Pipelines are not available on CircleCI server v2.x._
```
Comment (catherinepope) [inline jekyll/_cci2/glossary.adoc:68]:
```suggestion
A <<pipelines#,CircleCI pipeline>> is the full set of processes you run when you trigger work on your projects. Pipelines encompass your workflows, which in turn coordinate your jobs. Pipelines are defined in your project '.circleci/config.yml.' file. _Pipelines are not available on CircleCI server v2.x._
```

**Review by catherinepope — APPROVED** [review]
👍  Just spotted one missing _l_.

**Comment by devinmarieb** [comment]
@rsylvian Awesome thanks! I'm on vacation next week, so if it does get attention next week, feel free to merge it/have someone else merge it 👍

**Pull request merged**